tests/performance_tests/performance_test_basic.py

```diff
@@ -436,6 +450,8 @@ def analyzeResultsAndReport(self, testResult: PtbTpsTestResult):
                                                completedRun=testResult.completedRun)
 
         jsonReport = None
+        self.ptbConfig.quiet = True
+        self.ptbConfig.delReport = True
```

Comment:
Why are these hardcoded now?

Reply:
Something in the report isn't properly being converted in json.dumps. I'll need to hunt down the reason.